.github/workflows/build_release_candidate.yml

```diff
@@ -109,7 +109,7 @@ jobs:
             RC_ZIP="${RC_DIR}.zip"
             RELEASE_DIR="beam-${{ github.event.inputs.RELEASE }}"
             RC_TAG="v${{ github.event.inputs.RELEASE }}-RC${{ github.event.inputs.RC }}"
-            SOURCE_RELEASE_ZIP="apache-beam-${RELEASE}-source-release.zip"
+            SOURCE_RELEASE_ZIP="apache-beam-${{ github.event.inputs.RELEASE }}-source-release.zip"
```

Comment:
otherwise the name appears to be "apache-beam--source-release.zip" (should be apache-beam-2.49.0-source-release.zip) see https://dist.apache.org/repos/dist/dev/beam/2.49.0/